.github/renovate.json5

```
@@ -1,6 +1,6 @@
 {
   "extends": [
     "github>int128/renovate-base",
-    "github>int128/go-actions",
+    "github>int128/go-renovate-config:github-actions#v1.2.0",
```

https://github.com/int128/go-renovate-config/releases/tag/v1.4.1

```suggestion
    "github>int128/go-renovate-config:github-actions#v1.4.1",
```